ansible/roles/openhpc_runtime/tasks/runtime.yml

```diff
@@ -1,6 +1,5 @@
 ---
 - name: Ensure selected OpenHPC packages are installed
   yum:
-    name: "{{item}}"
+    name: "{{openhpc_packages}}"
```

Comment:
This should have no effect - the with_items is translated to a single list for the yum module.

Reply:
I'll take your word for it.